sale_timesheet_limit_date/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,5 @@
+This module adds a date field to sale order and invoice. This date limits the
+selection of timesheet lines to include them in sale order and invoice.
+When an invoice is created from a sale order, the limit date will be propagated from the sale order to the invoice,
+and the date in sale order is removed.
```

Comment:
Are there known limitation like if I create a second invoice if there is still one in draft?

Reply:
in v12 lines connected to invoice when it created, in draft state, so if you have 2 linked invoices each one will keep his lines separately,  validation have no effect 
AFAIS the only option to union them in one invoice is remove invoices and create new one 
and now I think that limit date in invoice is redundant, but not blocking